### DISCUSSION TITLE
Microsoft's Virtual Filesystem for Git

### DISCUSSION THREAD

**Discussion by The-Futurist** [community/community#21713] [New to GitHub]
<p>Is anyone here using this? have you encountered problems? how did you install it? what did you install? </p>
<p>If I go here:</p>
<p><a href="https://vfsforgit.org/" rel="nofollow noopener">https://vfsforgit.org/</a></p>
<p>then click “GET STARTED” (on the left of the page) it takes me here:</p>
<p><a href="https://github.com/Microsoft/VFSForGit" rel="nofollow noopener">https://github.com/Microsoft/VFSForGit</a></p>
<p>I then eventually get to here:</p>
<p><a href="https://github.com/microsoft/VFSForGit/releases/tag/v1.0.19140.2" rel="nofollow noopener">https://github.com/microsoft/VFSForGit/releases/tag/v1.0.19140.2</a></p>
<p>My question is can I simply click that and install? is there any risk? is this product regarded as “production ready” ?</p>
<p>Ever since MS announced this I’ve had trouble getting 100% clarity on whether it’s officially “released” or why isn’t it installed in Windows by default (sure not everyone will use it but at least we’d be confident it was a solid trusted part of the OS).</p>
<p>Since Microsoft are now the parent company I’d expect that GitHub should be the goto place for detailed information and advice on this.</p>
<p>Any info much appreciated.</p>
<p>Thanks</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>While Microsoft did acquire GitHub, the two companies operate independently of each other. Because of this, we don’t have any information to share about Microsoft’s VFS for Git project. You’ll have to reach out to Microsoft for support of or information on that project.</p>
